test/expected/out_symlinked_dirs_osx.txt

```diff
@@ -1,5 +1 @@
 include
-
-aaa/include:
```

Comment:
Does this mean these headers aren't made being made available after this change? Can you explain why this changed?